### ISSUE TITLE
mdi-plus-box-multiple and mdi-plus-box-multiple-outline missing

### ISSUE THREAD

**Issue by lingtjien** [Templarian/MaterialDesign#4422]
I use these icons and after updating today they disappeared. Then I went searching inside the `@mdi/font/css/materialdesignicons.css`. Only finding `.mdi-numeric-9-plus-box-multiple` and `.mdi-numeric-9-plus-box-multiple-outline` defined, but I was expecting to find two more. `.mdi-plus-box-multiple` and `.mdi-plus-box-multiple-outline`

**Comment by PeterShaggyNoble** [comment]
Please ensure you are using the latest version of MDI.

- `plus-box-multiple` was previously `library-plus` and was renamed in v4.1.95
- `plus-box-multiple-outline` was added in the recently released v4.4.95

**Comment by lingtjien** [comment]
yeah I just discovered that a lot of things are wrong with my version upgrade, so it's probably not this package that's causing it. Thank you for getting back to me so fast!
